.github/workflows/tests.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - name: Download test data
       run: |
-        gdown https://drive.google.com/uc?id=1SyHRaJNvVa7V08mw-4_Vqj7tCynRRA3x
+        gdown https://drive.google.com/uc?id=1a0EA7sH_m0m9c1127Geq_bNnDmi7nj3x
```

Comment:
Why was this changed?

Reply:
Previous blob mapping table csv file's column is vector_id, I changed it to __vector_id. And I also added a law dataset.